### DISCUSSION TITLE
Please explain difference between queue and playlist

### DISCUSSION THREAD

**Discussion by ustraub** [jcorporation/myMPD#82] [Q&A] [answered]
Hi,
I do not understand the difference between a queue and a playlist. Obvioulsy both concepts implement the idea of a virtual list of songs, but what is the difference?

**Comment by jcorporation** [comment, marked as answer]
See https://www.musicpd.org/doc/html/user.html

> The queue (sometimes called “current playlist”) is a list of songs to be played by MPD. To play a song, add it to the queue and start playback.

> Stored playlists are some kind of secondary playlists which can be created, saved, edited and deleted by the client. They are addressed by their names. Its contents can be loaded into the queue, to be played back.
